docs/IE_DG/Extensibility_DG/AddingNGraphOps.md

```diff
@@ -1,61 +1,63 @@
 # Custom nGraph Operation {#openvino_docs_IE_DG_Extensibility_DG_AddingNGraphOps}
 
-Inference Engine Extension API allows to register operation sets (opsets) with custom nGraph operations, it allows to support Networks with unknown operations.
+Inference Engine Extension API enables you to register operation sets (opsets) with custom nGraph operations and to support Networks with unknown operations.
 
 ## Operation Class
 
 To add your custom nGraph operation, create a new class that extends `ngraph::Op`, which is in turn derived from `ngraph::Node`, the base class for all graph operations in nGraph. Follow the steps below:
 
 1. Define a `NodeTypeInfo` object that identifies the type of the operation to the graph users and helps with dynamic type resolution. The type info of an nGraph operation currently consists of a string identifier and a version number, but this may change in the future.
 
-2. Implement constructors that can optionally take the operation inputs and attributes as parameters. 
+2. Implement constructors that optionally take the operation inputs and attributes as parameters. 
 
-3. Override the shape inference method `validate_and_infer_types`. This method is called multiple times during graph manipulations to determine the shapes and element types of the outputs of the operations. You can access the input shapes through the `get_input_partial_shape()` method and input element types through the `get_input_element_type()` method of `ngraph::Node`. Set the inferred shape and element type of the output using `set_output_type`.
+3. Override the shape inference method `validate_and_infer_types`. This method is called multiple times during graph manipulations to determine the shapes and element types of the outputs of the operations. To access the input shapes and input element types, use the `get_input_partial_shape()` and `get_input_element_type()` methods of `ngraph::Node`. Set the inferred shape and element type of the output using `set_output_type`.
```

Comment:
`...of the outputs of the operations.` -> `of the operations outputs.`?

Reply:
"using `set_output_type`." -> "using `set_output_type` method."?

Reply:
Fixed the 1st one, the 2nd one is ok